yt/visualization/tests/test_geo_projections.py

```diff
@@ -151,6 +161,21 @@ def test_nondefault_transform(self):
                 self.slc.plots[("stream", "Density")].axes.projection, proj_type
             )
 
+    @requires_module("cartopy")
+    def test_extent(self):
+        # checks that the axis extent is narrowed when doing a subselection
+        axis = "altitude"
+        slc = yt.SlicePlot(self.ds, axis, ("stream", "Density"), origin="native")
```

Comment:
just a thought here, as I see `origin="native"` being used for every single slice plot in this test module. Does it make sense to use any other value ? if not, then it's something I missed in https://github.com/yt-project/yt/pull/3863, and could easily be corrected (not necessarily in this PR though)

Reply:
Don't think you missed anything -- maybe at one point we didn't have any of the validation to set `origin="native"`? I haven't noticed any problems when omitting the `origin` kwarg. I did remember this morning that I wanted to add a test for a dataset that is not global (I only added a test for plotting a subset of a global dataset). Maybe I'll leave out the kwarg from that one...